src/System.Data.SqlClient/src/ApiCompatBaseline.net46.txt

```diff
@@ -0,0 +1,7 @@
+MembersMustExist : Member 'System.Data.SqlClient.SqlBulkCopy.WriteToServer(System.Data.Common.DbDataReader)' does not exist in the implementation but it does exist in the contract.
```

Comment:
Good, we'll need @saurabh500 to comment if these contracts need to work on 4.5 or if they can be moved to 4.5.1.